builtin/mainmenu/tab_settings_simple.lua

```diff
@@ -90,67 +90,68 @@ local languages, language_dropdown, lang_idx = get_language_list()
 	local sound = tonumber(core.settings:get("sound_volume")) ~= 0 and true or false
 
 	local tab_string =
-		"box[-0.1,0;3.85,5.5;#999999]" ..
-		"checkbox[0.15,-0.05;cb_smooth_lighting;" .. fgettext("Smooth Lighting") .. ";"
+		"background9[0,0.05;3.85,5.5;" .. defaulttexturedir_esc .. "desc_bg.png;false;32]" ..
+		"checkbox[0.15,-0.1;cb_smooth_lighting;" .. fgettext("Smooth Lighting") .. ";"
 				.. dump(core.settings:get_bool("smooth_lighting")) .. "]" ..
-		"checkbox[0.15,0.5;cb_particles;" .. fgettext("Particles") .. ";"
+		"checkbox[0.15,0.45;cb_particles;" .. fgettext("Particles") .. ";"
 				.. dump(core.settings:get_bool("enable_particles")) .. "]" ..
-		"checkbox[0.15,1.1;cb_3d_clouds;" .. fgettext("3D Clouds") .. ";"
+		"checkbox[0.15,1.05;cb_3d_clouds;" .. fgettext("3D Clouds") .. ";"
 				.. dump(core.settings:get_bool("enable_3d_clouds")) .. "]" ..
-	--[["checkbox[0.15,1.7;cb_opaque_water;" .. fgettext("Opaque Water") .. ";"
+	--[["checkbox[0.15,1.65;cb_opaque_water;" .. fgettext("Opaque Water") .. ";"
 				.. dump(core.settings:get_bool("opaque_water")) .. "]" ..
-		"checkbox[0.15,2.0;cb_connected_glass;" .. fgettext("Connected Glass") .. ";"
+		"checkbox[0.15,1.95;cb_connected_glass;" .. fgettext("Connected Glass") .. ";"
 				.. dump(core.settings:get_bool("connected_glass")) .. "]" ..]]
-		"checkbox[0.15,1.7;cb_fog;" .. fgettext("Fog") .. ";"
+		"checkbox[0.15,1.65;cb_fog;" .. fgettext("Fog") .. ";"
 				.. dump(core.settings:get_bool("enable_fog")) .. "]" ..
-		"checkbox[0.15,2.3;cb_inventory_items_animations;" .. fgettext("Inv. animations") .. ";"
+		"checkbox[0.15,2.25;cb_inventory_items_animations;" .. fgettext("Inv. animations") .. ";"
 				.. dump(core.settings:get_bool("inventory_items_animations")) .. "]" ..
-		"checkbox[0.15,2.9;cb_fancy_leaves;" .. fgettext("Fancy Leaves") .. ";"
+		"checkbox[0.15,2.85;cb_fancy_leaves;" .. fgettext("Fancy Leaves") .. ";"
 				.. dump(fancy_leaves) .. "]" ..
-		"checkbox[0.15,3.5;cb_touchtarget;" .. fgettext("Touchtarget") .. ";"
-				.. dump(touchtarget) .. "]" ..
-		"checkbox[0.15,4.1;cb_arm_inertia;" .. fgettext("Arm inertia") .. ";"
+		"checkbox[0.15,3.45;cb_touchtarget;" .. fgettext("Crosshair") .. ";"
+				.. dump(not touchtarget) .. "]" ..
+		"checkbox[0.15,4.05;cb_arm_inertia;" .. fgettext("Arm inertia") .. ";"
 			.. dump(arm_inertia) .. "]" ..
-		"checkbox[0.15,4.7;cb_sound;" .. fgettext("Sound") .. ";"
+		"checkbox[0.15,4.65;cb_sound;" .. fgettext("Sound") .. ";"
 				.. dump(sound) .. "]" ..
 
-		"box[4,0;3.75,5.5;#999999]" ..
+		"background9[4.1,0.05;3.75,5.5;" .. defaulttexturedir_esc .. "desc_bg.png;false;32]" ..
 
-		"label[4.25,0.15;" .. fgettext("Maximum FPS") .. ":]" ..
-		"dropdown[4.25,0.6;3.5;dd_fps_max;30,35,45,60;" ..
-			tonumber(fps <= 30 and 1 or fps == 35 and 2 or fps == 45 and 3 or 4) .. "]" ..
+		"label[4.25,0.1;" .. fgettext("Maximum FPS") .. ":]" ..
+		"dropdown[4.25,0.55;3.5;dd_fps_max;30,35,45,60,90;" ..
```

Comment:
@luk3yx dropdown in FS3 can use values instead if ID?

Reply:
Dropdowns have an `index_event` parameter (false by default in all formspec versions) which will make them use a number instead of the value.

Reply:
Would it be more convenient to use in this tab?